### PR TITLE
Add an angles version of zero vector and identity matrix

### PR DESCRIPTION
**Pull request by JohnAFernandez**
Quick pull request to add angles to the empty vectors and identity matrix party.  I wanted to do it in the the multi branch, but I realized it was better to keep changes like this out of the way of multi getting done.

### CODE REVIEW

**Comment by z64555** [comment]
Not really an improvement if this PR is just a declaration of an unused constant.  If the PR included places where the constants could be used, then I'd give the OK.

**Comment by JohnAFernandez** [comment]
I have this code block in the testnetworking branch that looks like this without it:

`temp_interp.old_angles = { 0.0f, 0.0f, 0.0f };`
`temp_interp.new_angles = { 0.0f, 0.0f, 0.0f };`
`temp_interp.anticipated_angles_a = { 0.0f, 0.0f, 0.0f };`
`temp_interp.anticipated_angles_b = { 0.0f, 0.0f, 0.0f };`
`temp_interp.orientation_error = { 0.0f, 0.0f, 0.0f };`

But I can go hunt for some use cases, if you want.

**Comment by z64555** [comment]
Yes, please.

**Comment by JohnAFernandez** [comment]
Found three use cases.

File: code/ship/shipfx.cpp [hunk 1]
```diff
@@ -106,7 +106,7 @@ static void shipfx_subsystem_maybe_create_live_debris(object *ship_objp, ship *s
 
 	// copy angles
 	angles copy_angs = pm->submodel[submodel_num].angs;
-	angles zero_angs = {0.0f, 0.0f, 0.0f};
+	angles zero_angs = vmd_zero_angles;
```
Comment (z64555) [inline code/ship/shipfx.cpp:109]: Delete this line and plug in vmd_zero_angles directly into line 145
`pm->submodel[submodel_num].angs = zero_angs;`

**Comment by JohnAFernandez** [comment]
That's what I get for looking at 1 in the morning.

File: code/ship/shipfx.cpp (hunk 1 above)
Comment (JohnAFernandez) [reply code/ship/shipfx.cpp]: Everything should be fixed now!

**Pull request merged**